core/templates/dev/head/css/oppia.css

```diff
@@ -3683,3 +3683,40 @@ md-card.preview-conversation-skin-supplemental-card {
   padding: 0;
   width: 1px;
 }
+
+.hexagon {
+  transform: scale(0.5);
+  position: relative;
+  width: 200px;
+  height: 115.47px;
+  background-color: #009A8B;
+  margin: 20px 0;
+  border-left: solid 2px #009A8B;
+  border-right: solid 2px #009A8B;
+}
+
+.hexagon:before,
+.hexagon:after {
+  content: "";
+  position: absolute;
+  z-index: 1;
+  width: 141.42px;
+  height: 141.42px;
+  -webkit-transform: scaleY(0.5774) rotate(-45deg);
+  -ms-transform: scaleY(0.5774) rotate(-45deg);
+  transform: scaleY(0.5774) rotate(-45deg);
+  background-color: inherit;
+  left: 27.2893px;
+}
+
+.hexagon:before {
+  top: -70.7107px;
+  border-top: solid 2.8284px #009A8B;
+  border-right: solid 2.8284px #009A8B;
+}
+
+.hexagon:after {
+  bottom: -70.7107px;
```

Comment:
Add some comments on how you generated these values in case we ever need to change them in the future?

Reply:
Done.